Source/Core/DolphinWX/Debugger/RegisterView.cpp

```diff
@@ -324,20 +327,31 @@ wxString CRegTable::GetValue(int row, int col)
                                                    << 32 |
                                                PowerPC::ppcState.spr[SPR_IBAT4L + (row - 16) * 2]);
 
+      if (row == 16)
+        return wxString::Format("%016llx", (u64)PowerPC::ppcState.spr[SPR_TU]
```

Comment:
Sorry for missing this earlier, but `(u64)` should be `static_cast<u64>(...)` instead

Reply:
Shouldn't ```PRIx64``` be used as well?

Reply:
`(u64)` and `llx` is used in the other print statements. Should I just fix my addition or should I fix all of them?